content/posts/ai-crawlers-abuse.md

```diff
@@ -0,0 +1,177 @@
+title: AI Crawlers need to be more respectful
+date: 2024-07-25
+category: Engineering
+tags: ai, crawlers, abuse
+authors: Eric Holscher
+status: published
+image:
+image_credit:
+
+Many AI companies are starting to crawl many pages on the web,
```

Comment:
So, overall, I think this post would benefit from describing the problem we're facing sooner. The end of this post did a great job of that and especially describing our frustration.

Just tossing out an example, but maybe this first sentence just gets right to the point:

```suggestion
In the last few months, we have noticed an increase in abusive site crawling,
mainly from AI products and services. These products are recklessly crawling
many sites across the web, and we've already had to block several sources of abusive traffic.
```